observability/logging/logging-6.0/log6x-about.adoc

```diff
@@ -5,3 +5,46 @@ include::_attributes/common-attributes.adoc[]
 :context: logging-6x
 
 toc::[]
+
+The `ClusterLogForwarder` custom resource is the central configuration point for OpenShift Logging 6.0. It defines the desired state of your logging system, including inputs, outputs, and pipelines. The Cluster Logging Operator watches for changes to this resource and automatically applies the necessary configurations to the underlying logging components.
+
+== Service account
+This release of Cluster Logging requires administrators to explicitly grant log collection permissions to the service account associated with *ClusterLogForwarder*. This was not required in previous releases for the legacy logging scenario consisting of a *ClusterLogging* and, optionally, a *ClusterLogForwarder.logging.openshift.io* resource.
+
+To use the existing service account (that is, `logcollector`) from a previous release, create the following *ClusterRoleBinding*:
```

Comment:
This serviceAccount only exists in the namespace `openshift-logging`, then, if the clusterLogForwarder is created in a different namespace, it won't exist this serviceAccount failing to run.

Also, this is missing if it's created a new serviceAccount, then, it's needed to assign what log_types is allowed to collect. Then, it should be explained if it's not in a different place what clusterroles exists as indicated in https://docs.openshift.com/container-platform/4.16/observability/logging/log_collection_forwarding/log-forwarding.html#log-collection-rbac-permissions_log-forwarding